docs/content/Getting-Started/quickstart.md

```diff
@@ -35,7 +35,7 @@ This guide is intended to show how to (1) quickly bring up a **KubeStellar** env
 KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
 
 ```shell
-bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/bootstrap/bootstrap-kubestellar.sh) --kubestellar-version latest
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/{{ config.ks_branch }}/bootstrap/bootstrap-kubestellar.sh --kubestellar-version {{ config.ks_tag }}
```

Comment:
I think I see a typo, accidental removal of the closing paren around the curl command.

Reply:
Fixed